### ISSUE TITLE
Crash Log

### ISSUE THREAD

**Issue by StarcallerSolsti** [Soreepeong/XivAlexander#479]
[XivAlexander_20220715_205529_564.log](https://github.com/Soreepeong/XivAlexander/files/9125044/XivAlexander_20220715_205529_564.log)
![Crash Report](https://user-images.githubusercontent.com/106460088/179336632-2da5a4de-b773-4e92-8710-3cbb52cf4384.JPG)

**Comment by Llendra** [comment]
I'm having the same problem. https://i.imgur.com/vmQWkfl.png

**Comment by Haxarus** [comment]
I ve the same issue

**Comment by yjywybb** [comment]
I ve the same issue

**Comment by lunabor85** [comment]
Same issue here 

[XivAlexander_20220723_123929_577.log](https://github.com/Soreepeong/XivAlexander/files/9174091/XivAlexander_20220723_123929_577.log)

![image](https://user-images.githubusercontent.com/80270601/180612169-a0546671-74ed-4c12-b101-a25a184028b0.png)

**Comment by Actual** [comment]
Hey guys

this is a new error caused when your ff client isnt running with admin permissions. Doing this fixed it for me but leads to things like discord overlays not working

**Comment by Legit-Gou** [comment]
I am seeing the same issue, now constantly. At least 1 crash every hour up to about every 15 min. I do run FFXIV in admin mode too, that makes no difference
[XivAlexander_20220728_022951_739.log](https://github.com/Soreepeong/XivAlexander/files/9202708/XivAlexander_20220728_022951_739.log)
.

**Comment by Legit-Gou** [comment]
> Hey guys
> 
> this is a new error caused when your ff client isnt running with admin permissions. Doing this fixed it for me but leads to things like discord overlays not working

No, I run it in admin mode for other reason and still this has been happening.

**Comment by StarcallerSolsti** [comment]
Bump because this is ongoing and I always run in admin mode

**Comment by r3bychan** [comment]
Went on textools , disabled Bibo+ , launched the game , used bibo+ on penumbra , game crashed with same errors while browsing my glamour plates. Disabled penumbra , still crashes . Re-enabled Bibo+ on textools , all works. Even using penumbra on top of the bibo+ from textools works , no crashes. I dont run the game in admin.
